docs/.vitepress/vitepress/components/common/vp-theme-toggler.vue

```diff
@@ -61,7 +64,7 @@ const beforeChange = (): Promise<boolean> => {
 </script>
 
 <template>
-  <div @click.stop="switchTheme">
+  <div @click.stop="switchTheme" @keydown.enter.stop="handleKeyEnter">
```

Comment:
It should be possible to reuse `switchTheme` directly here.

Reply:
> It should be possible to reuse `switchTheme` directly here.

Considered reusing `switchTheme` directly, but `switchTheme` event parameter is `MouseEvent`. 
That include the coordinates of clicking mouse and generate transition based on position of the coordinates.
And `keydown.enter` is an inappropriate `KeyboardEvent`.

Reply:
You can expand the `switchTheme` function. The keyboard switching theme you currently implement does not apply the transition effect.